### PR TITLE
chore: minor fixes | /workflows/cd

### PR DESCRIPTION
**Pull request by dOrgJelli**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,71 @@
...
+let result = web_scraper.get_text(
+    &WebScraperArgsGetText{
+        url: "..."
+    }
+).unwrap();
+```
+
+`app/kotlin`:
+```kotlin
+val web_scraper = WebScraper(client)
```
Comment (krisbitney) [inline README.md]: nit: in Kotlin, the standard practice is to use camelCase

File: polywrap.graphql [hunk 2]
```diff
@@ -2,5 +2,5 @@
 
 type Module {
   get_text(url: String!): String!
-  get_links(uri: String!): String!
+  get_links(url: String!): String!
```
Comment (krisbitney) [inline polywrap.graphql:5]: This is a nice change!

File: src/__tests__/e2e/integration.spec.ts [hunk 3]
```diff
@@ -1,45 +1,49 @@
...
-  it("calls get_text", async () => {
-    const url = "https://polywrap.io";
-
-    const result = await client.invoke<App.Module_GetText>({
-      uri: wrapperUri,
-      method: "get_text",
-      args: { url: url }
+  it("get_text", async () => {
+    const result = await webScraper.get_text({
+      url: "\nhttps://webscraper.io/test-sites/e-commerce/allinone\n"
```
Comment (krisbitney) [inline src/__tests__/e2e/integration.spec.ts:41]: why does the url need newlines?

**Review by krisbitney — APPROVED** [review]
Thanks for cleaning up this wrap!

File: src/__tests__/e2e/integration.spec.ts (hunk 3 above)
Comment (dOrgJelli) [reply src/__tests__/e2e/integration.spec.ts:41]: It's just to make sure that the links are being "split" by `\n`. We can also add a feature where the user defines what the text delimiter is.

I think `\n` is a better default than ` `

Comment (krisbitney) [reply src/__tests__/e2e/integration.spec.ts:41]: That change is great, but are the `\n` needed in the method argument?

Comment (krisbitney) [reply src/__tests__/e2e/integration.spec.ts:41]: I think this might be a typo

**Pull request merged**